common/utils.go

```diff
@@ -53,6 +64,28 @@ func TarantoolIsEnterprise(tarantoolDir string) (bool, error) {
 	return strings.HasPrefix(tarantoolVersion, "Tarantool Enterprise"), nil
 }
 
+// TarantoolVersion gets Tarantool version
+func GetTarantoolVersion(tarantoolDir string) (string, error) {
+	var err error
+
+	tarantool := filepath.Join(tarantoolDir, "tarantool")
+	versionCmd := exec.Command(tarantool, "--version")
+
+	tarantoolVersion, err := GetOutput(versionCmd, nil)
+	if err != nil {
+		return "", err
+	}
+
+	r := regexp.MustCompile(`\d+\.\d+\.\d+-\d+-\w+`)
```

Comment:
If you plan to call this function more the one time at application execution, it could be reasonable to compile regexp into global var (not for every call). But feel free to ignore. Anyway it shouldn't lead to dramatic performance degradation for user application.

So you also could make variable name more expressive.